source/bricklaunch

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+# bricklaunch.sh <ops> <program>
+# - open a new vt and switch to it
+# - give 'user' ownership of the vt
+# - run the given programm on the new vt as 'user'
+#
+
+OPTIND=1
+user='user'
```

Comment:
That is what we suggest here already:
http://www.ev3dev.org/docs/getting-started/#set-up-a-new-user